app/src/main/java/ar/com/wolox/android/example/network/repository/UserRepository.kt

```diff
@@ -14,6 +14,7 @@ class UserRepository @Inject constructor(private val retrofitServices: RetrofitS
         get() = retrofitServices.getService(UserService::class.java)
 
     suspend fun authUser(email: String, password: String) = withContext(Dispatchers.IO) {
-        NetworkRequestHandler.safeApiCall { service.authUser(UserAuth(email, password)) }
+        val response = NetworkRequestHandler.safeApiCall { service.authUser(UserAuth(email, password)) }
+        response
```

Comment:
Since the last line is only `response` you can ommit the val declaration and call just `NetworkRequestHandler.safeApiCall { service.authUser(UserAuth(email, password)) }`